src/main/index.js

```diff
@@ -225,6 +225,40 @@ function runApp() {
     }
   })
 
+  // If in windows, add Windows Jump List task 'New Window'
```

Comment:
Please remove this code comment, it doesn't add any additional information compared to what the code already says (adding code comments is generally only necessary if the code alone isn't enough, e.g. explaining why some logic is there).